evm/test/NttManager.t.sol

```diff
@@ -72,12 +72,20 @@ contract TestNttManager is Test, IRateLimiterEvents {
 
     // === pure unit tests
 
-    function test_countSetBits() public {
-        assertEq(countSetBits(5), 2);
-        assertEq(countSetBits(0), 0);
-        assertEq(countSetBits(15), 4);
-        assertEq(countSetBits(16), 1);
-        assertEq(countSetBits(65535), 16);
+    // naive implementation of countSetBits to test against
+    function simpleCount(uint64 n) public pure returns (uint8) {
```

Comment:
For what it's worth, I tested the 'countSetBits()' in Python using the implementation and feel pretty good about it as well.

Reply:
tested just in case